README.md

```diff
@@ -27,6 +27,20 @@ Note: `src/backend/sinr` is a submodule from `UMassCDS/inatrualist-sinr`, if you
 
 Note: If you need to switch to a particular `UMassCDS/inaturalist-sinr` branch and run the prototype, navigate to `src/backend/sinr`, use `git checkout <branch-name>` to switch to a branch. Now the submodule will be at a different branch.
 
+## Working with database
+
+1. Notice that there are two .env files, .docker.env and .env, .env is used for your local development, .docker.env will be used in your containers.
```

Comment:
This can be less redundant. Maybe "Notice that there are two .env files -- used for your local development-- and .docker.env -- used for your containers/"

This shouldn't be listed as 1., since you don't really need to take action. Also might be good to explain why the containers AND local are needed at the same time.